system-variables.md

```diff
@@ -466,21 +497,24 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 ### tidb_auto_analyze_start_time
 
 - Scope: GLOBAL
+- Persists to cluster: Yes
 - Default value: `00:00 +0000`
 - This variable is used to restrict the time window that the automatic update of statistics is permitted. For example, to only allow automatic statistics updates between 1AM and 3AM, set `tidb_auto_analyze_start_time='01:00 +0000'` and `tidb_auto_analyze_end_time='03:00 +0000'`.
 
 ### tidb_backoff_lock_fast
 
 - Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
 - Default value: `10`
 - Range: `[1, 2147483647]`
 - This variable is used to set the `backoff` time when the read request meets a lock.
 
 ### tidb_backoff_weight
 
 - Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
 - Default value: `2`
-- Range: `[1, 2147483647]`
+- Range: `[0, 2147483647]`
```

Comment:
This is from https://github.com/pingcap/tidb/pull/30664 - it only affects master and not TIDB 6.0.